mitosheet/mitosheet/saved_analyses/step_upgraders/set_column_formula.py

```diff
@@ -42,3 +42,40 @@ def upgrade_set_column_formula_1_to_2(step: Dict[str, Any], later_steps: List[Di
         "step_type": "set_column_formula", 
         "params": params
     }] + later_steps
+    
+def upgrade_set_column_formula_2_to_3(step: Dict[str, Any], later_steps: List[Dict[str, Any]]) -> List[Dict[str, Any]]:
+    """
+    Adds in the formula_label, which defaults to 0 for all upgrades. This is used for calculating offsets 
+    when users are writing rolling window formulas.
+
+    OLD: {
+        'step_version': 2, 
+        'step_type': "set_column_formula",
+        'params': {
+            sheet_index: 0,
+            column_id: _column id_,
+            old_formula: '=A',
+            new_formula: '=B',
+        }
+    }
+
+    NEW: {
+        'step_version': 3, 
+        'step_type': "set_column_formula",
+        'params': {
+            sheet_index: 0,
+            column_id: _column id_,
+            formula_label: 0,
+            old_formula: '=A',
+            new_formula: '=B',
+        }
+    }
+    """
+    params = step['params']
+    params['formula_label'] = 0
```

Comment:
This is fine because none of the formulas have an offset in it. They are all column headers without indexes, so we won't calculate any offsets. 

Is the fact that the `0` might not be an index in the dataframe an issue?